airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV1.py

```diff
@@ -15,24 +11,24 @@
 
 
 class ConnectorType(Enum):
-    destination = "destination"
-    source = "source"
+    destination = 'destination'
```

Comment:
We generally use double quotes for python strings.

Reply:
Weird, sorry this was a generated file. Not sure why it changed. Ill look into it